File: VeamIt-project/iOS/veam00000000/veam00000000/TemplateForum.m
//
//  TemplateForum.m
//  veam00000000
//
//  Created by veam on 6/4/14.
//  Copyright (c) 2014 veam. All rights reserved.
//

#import "TemplateForum.h"

@implementation TemplateForum

@synthesize templateForumId ;
@synthesize title ;

- (id)initWithAttributes:(NSDictionary *)attributeDict
{
    self = [super init] ;
    
    [self setTemplateForumId:[attributeDict objectForKey:@"id"]] ;
    [self setTitle:[attributeDict objectForKey:@"t"]] ;
    
    return self ;
}

@end

File: VeamIt-project/Android/veam31000287/veam31000287/src/main/java/co/veam/veam31000287/TemplateForum.java
package co.veam.veam31000287;

import org.xml.sax.Attributes;

/**
 * Created by veam on 10/27/16.
 */
public class TemplateForum {
    private String templateForumId ;
    private String title ;

    public TemplateForum() {

    }

    public String getTemplateForumId() {
        return templateForumId;
    }

    public void setTemplateForumId(String templateForumId) {
        this.templateForumId = templateForumId;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public TemplateForum(Attributes attributes) {
        this.setTemplateForumId(attributes.getValue("id")) ;
        this.setTitle(attributes.getValue("t")) ;
    }
}
